CHANGELOG.rst

```diff
@@ -2,6 +2,17 @@
 Changelog for package udp_driver
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.1.0 (2021-02-18)
+------------------
+* Tracking PR: https://github.com/ros-drivers/transport_drivers/pull/31
+* Implemented IoContext (The main I/O Context that can be configured as single or multi thread with async task execution support)
+* Implemented UdpSocket (The UDP socket functionality that working in blocking and non-blocking (Async with task execution) modes)
+* Implemented UdpDriver Facade (Facade design pattern implementation that acts as a sender and receiver UDP socket pair container)
+* Implemented ROS2 and raw buffer message converters for Int, UInt and Float variants in std_msgs namespace.
+* Implemented UdpDriverNode example and its test.
+* Implemented IoContext, UdpSocket and UdpDriver tests.
+* Contributors: Reza Ebrahimi
+
```

Comment:
This is not the purpose of the CHANGELOG. Your changes do not constitute a release. Please remove these changes.

Reply:
What is the purpose of change log? Why I should remove contributors part from change log?

Reply:
If your concern is release version, We can change it. But I didn't understand why I should remove the implemented API log and contributors part.

Reply:
What is the purpose of change log? If your concern is release version, We can change it. But I didn't understand why I should remove the implemented API log and contributors part.

Reply:
Changelogs are automatically generated as part of the release process. Same for version increments.

@JWhitleyWork is correct in that it's not necessary (nor perhaps desirable) to change this file by hand before running the tools.

The suggestion is actually to revert all your changes to the `CHANGELOG.rst` file.

Reply:
OK. I got it, will revert it.

Reply:
Done with 9a82e24abe471c61021e8cd86e3f734dbff0b123.